backend/controller/console.go

```diff
@@ -40,6 +40,40 @@ func (*ConsoleService) Ping(context.Context, *connect.Request[ftlv1.PingRequest]
 	return connect.NewResponse(&ftlv1.PingResponse{}), nil
 }
 
+func visitNode(sch *schema.Schema, n schema.Node, verbString *string) error {
+	return schema.Visit(n, func(n schema.Node, next func() error) error {
+		switch n := n.(type) {
+		case *schema.Ref:
+			data, err := sch.ResolveRefMonomorphised(n)
+			if err != nil {
+				return err
+			}
+			*verbString += data.String() + "\n\n"
+
+			err = visitNode(sch, data, verbString)
+			if err != nil {
+				return err
+			}
+		default:
+		}
+		return next()
+	})
+}
+
+func verbSchemaString(sch *schema.Schema, verb *schema.Verb) (string, error) {
+	var verbString string
+	err := visitNode(sch, verb.Request, &verbString)
+	if err != nil {
+		return "", err
+	}
+	err = visitNode(sch, verb.Response, &verbString)
+	if err != nil {
+		return "", err
+	}
+	verbString += verb.String()
+	return verbString, nil
+}
```

Comment:
@alecthomas is this a reasonable approach?

Reply:
What you had before was fine in terms of visiting all nodes recursively, but I don't think it follows `Ref`s under normal circumstances, which is something this will want to do.

Reply:
What I mean is if you have this:

```
data User {
  name String
}

data Req {
  user User
}
```

The "user" field will be represented as a `Ref` in the schema, and I don't think (could be wrong) that will get visited under normal circumstances.

Reply:
Yeah I think you're right. That's why I added that `visitNode` func so that I could visit the `Ref` as well. But maybe I should update the `schema.Visit` to follow these refs as well?

Reply:
I'll merge this PR for the console updates and fixes and we can sync on this when you have a sec. Want to make sure I get this updated as needed. 👍